src/routes/index.ts

```diff
@@ -18,3 +27,31 @@ export function applicationRoutes(app: Koa<any, {}>) {
   app.use(trackingPlanRoutes);
   app.use(userTransformRoutes);
 }
+
+export function addSwaggerRoutes(app: Koa<any, {}>) {
+  // Ading swagger routes, check swagger docs in http://localhost:5050/docs
+  if (enableSwagger) {
+    const router = new Router();
+    if (existsSync('./dist/src/swagger.json')) {
+      let rawContent: string | undefined;
+      try {
+        rawContent = readFileSync('./dist/src/swagger.json', { encoding: 'utf8' });
```

Comment:
Shouldn't we use the relative paths here rather than referring to the `dist` in the paths?
See how `features.json` is accessed.

Reply:
Done